components/ProtectedImage.vue

```diff
@@ -1,10 +1,16 @@
 <script lang="ts" setup>
 import { useAuth0 } from "@auth0/auth0-vue";
 
-const props = defineProps<{
-  src: string;
-  alt?: string;
-}>();
+const props = withDefaults(
+  defineProps<{
+    src: string;
+    alt?: string;
+    noBorder: boolean;
```

Comment:
If it is optional, it should be marked with a question mark here; otherwise the system thinks its required.

Reply:
done